.circleci/config.yml

```diff
@@ -96,21 +130,12 @@ jobs:
 
                docker push grafana/carbon-relay-ng:$tag
             fi
-workflows:
-  version: 2
-  pr_acceptance:
-    jobs:
-      - test
-      - build
-  build_accept_deploy:
-    jobs:
-      - test
-      - build
-      - package:
-          requires:
-            - build
-      - deploy:
-          requires:
-            - test
-            - build
-            - package
+
+  github_binaries:
+      docker:
+        - image: cimg/go:1.18
+      steps:
+        - checkout
+        - run: curl -sfL https://goreleaser.com/static/run | bash
+        # For dry runs use the following and comment the above
+        # - run: curl -sfL https://goreleaser.com/static/run | bash -s -- --rm-dist --skip-publish --snapshot
```

Comment:
Took me a few minutes to find out how to do a dry run so I think this comment adds value for future us if we run into any issue